themes/aquablank/layouts/partials/list_cloudsploit.html

```diff
@@ -35,6 +36,28 @@ <h1 class="title page_title is-spaced fadeInUp animationDelay_1">Remediation Gui
             </div><!-- column is-8 -->
 
             <div class="column is-4-desktop is-6-tablet is-12-mobile fadeInUp animationDelay_2">
+                <div class="avdcve_sidebar_wrap ">
+                    <div class="project_item">
+                        <h2 class="is-size-5">Browse other categories:</h2>
+                        <ul class="my-2">
+							<li class=""><a href="{{.Permalink}}">AWS</a> </li>
+							<li class=""><a href="{{.Permalink}}">GCP</a> </li>
+							<li class=""><a href="{{.Permalink}}">Docker</a> </li>
+							<li class=""><a href="{{.Permalink}}">Kubernetes</a> </li>
+							<li class=""><a href="{{.Permalink}}">Runtime Threats</a> </li>
+							<li class=""><a href="{{.Permalink}}">CVEs</a> </li>
+						</ul>
+					
+                    </div>
+                   
+                </div>
```

Comment:
Also don't work. They are all pointing to /cspm. What were you trying to implement here?